Automattic/jetpack
# Random Redirect: add deprecation warning

Fixes https://github.com/Automattic/vulcan/issues/352

## Proposed changes:

The Random Redirect module will soon be removed from the Jetpack plugin. This PR adds a deprecation warning in the codebase that will be triggered if the feature is used.

## Jetpack product discussion


Project thread: pfwV0U-bF-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Make sure you can see the server logs
- Install the Panel theme, or add `add_theme_support( 'jetpack-comic' )` to your theme
- Visit `/?random`
- You should see the following warning in the log: `PHP Deprecated:  File .../jetpack/modules/theme-tools/random-redirect.php is deprecated since version jetpack-$$next-version$$ with no alternative available. Starting Jetpack 13.6, Jetpack will no longer support the Random Redirect module. in ...`
